waiter/src/waiter/core.clj

```diff
@@ -74,17 +73,16 @@
             [waiter.websocket :as ws]
             [waiter.work-stealing :as work-stealing])
   (:import (java.net InetAddress URI)
-           java.util.concurrent.Executors
+           (java.util.concurrent Executors)
            (javax.servlet ServletRequest)
-           org.apache.curator.framework.CuratorFrameworkFactory
-           org.apache.curator.framework.api.CuratorEventType
-           org.apache.curator.framework.api.CuratorListener
-           org.apache.curator.framework.recipes.leader.LeaderLatch
-           org.apache.curator.retry.BoundedExponentialBackoffRetry
-           org.eclipse.jetty.client.HttpClient
-           org.eclipse.jetty.client.util.BasicAuthentication$BasicResult
-           org.eclipse.jetty.websocket.client.WebSocketClient
-           (org.eclipse.jetty.websocket.servlet ServletUpgradeResponse ServletUpgradeRequest)))
+           (org.apache.curator.framework.api CuratorEventType CuratorListener)
+           (org.apache.curator.framework CuratorFrameworkFactory)
+           (org.apache.curator.framework.recipes.leader LeaderLatch)
```

Comment:
This doesn't look right... The `framework` import should come before both `framework.*` imports.

Reply:
Updated, seems like a bug in Amethyst formatting :(